index.js

```diff
@@ -239,6 +264,39 @@ exports.appActivityAudit = function(isStart, sysname, hostname, username) {
 };
 
 /**
+ * Generates a 'Audit Log Used' audit message in XML format
+ * @param  {Number} outcome       the desired outcome, for authentication failure use atna.OUTCOME_MINOR_FAILURE.
```

Comment:
I think there is a copy-paste problem with this description, it shouldn't reference authentication failure.